packages/stops-overlay/src/StopsOverlay.story.js

```diff
@@ -44,23 +45,33 @@ class Example extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const { stops } = this.props;
+    this.setState({ stops });
+  };
+
   refreshStops = bounds => {
-    const stops = mockStops.filter(
+    const { stops, filterStops } = this.props;
```

Comment:
Sort props.

Reply:
Sorted in 26bfb195aac04d027a58cc3ccd29ee76ae6d4544